llama.h

```diff
@@ -59,9 +59,10 @@ extern "C" {
     typedef int32_t llama_seq_id;
 
     enum llama_vocab_type {
-        LLAMA_VOCAB_TYPE_SPM = 0, // SentencePiece
-        LLAMA_VOCAB_TYPE_BPE = 1, // Byte Pair Encoding
-        LLAMA_VOCAB_TYPE_WPM = 2, // WordPiece
+        LLAMA_VOCAB_TYPE_SPM      = 0, // SentencePiece
+        LLAMA_VOCAB_TYPE_BPE      = 1, // Byte Pair Encoding
+        LLAMA_VOCAB_TYPE_WPM      = 2, // WordPiece
+        LLAMA_VOCAB_TYPE_NO_VOCAB = 3, // For models without vocab
```

Comment:
Maybe we should call this LLAMA_VOCAB_TYPE_NONE instead.

Reply:
Used `LLAMA_VOCAB_TYPE_NONE = 0`. I hope I won't break compatibility.